### PR TITLE
Okapi wrapper updates to typescript

### LINKED ISSUE #332: Move okapi node/web entirely to typescript
Eliminate all the commonjs tests, replace with typescript.

### LINKED ISSUE #342: Support node development on Apple M1
This is currently blocked by the grpc tools not having precompiled grpc binaries for M1: https://github.com/grpc/grpc-node/issues/1405
Once this issue is complete, we can verify functionality.

### LINKED ISSUE #402: Support better typescript protobuf FFI interfaces.
Similar to some of the work done in the SDK, we need to switch okapi over to using `ts-proto`. This will enable the packages to be completely typescript in lieu of a mix of javascript and typescript.

### PR DESCRIPTION
**Pull request by fundthmcalculus**
Okapi package is now a single npm package with only the relevant wasm and js files bundled. This adds:

1. proper typescript support - interfaces over java-first method definitions
2. async/await support for node
3. testing node and browser as part of CI pipeline
4. tested against okapie-examples/node/basic-webpack to ensure proper functionality
5. incidentally, this also fixes the M1 protoc build issues.
6. Thanks :tm:  for pointing out the details of "browser" field.

closes #402
closes #332 
closes #342

### CODE REVIEW

**Comment by tmarkovski** [comment]
With the individual packages removed, we probably don't need the `lerna` tooling?

**Comment by fundthmcalculus** [comment]
Good points. I'll update that.

**Comment by fundthmcalculus** [comment]
> With the individual packages removed, we probably don't need the `lerna` tooling?

Lerna tooling removed.

File: wasm/packages/okapi/package.json [hunk 1]
```diff
@@ -1,46 +1,77 @@
 {
   "name": "@trinsic/okapi",
-  "version": "1.0.0-preview.27",
```
Comment (tmarkovski) [inline wasm/packages/okapi/package.json:3]: Can we revert this to simply 1.0.0? We're always overriding this when publishing, no need to specify actual production version here

Comment (fundthmcalculus) [reply wasm/packages/okapi/package.json:3]: I'm fine with `1.0.0`, will do. I just figured the `preview.27` was vastly out of date.

Comment (tmarkovski) [reply wasm/packages/okapi/package.json:3]: Yeah that was from the time we checked in the actual version and had no automation

**Comment by fundthmcalculus** [comment]
unfortunately, it doesn't look like this merge methodology will pan out. Closing in favor of a different PR.

**Review by tmarkovski — APPROVED** [review]

**Pull request merged**